migration/starter-for-10-PrintServiceAPI.yaml

```diff
@@ -1,3 +1,12 @@
+#
+# --------------------------------------------------------------------------------
+# This is the original starter-for-10 openAPI spec with initial API definitions
+# for Print Service API. As and when we design and implement each endpoint it
+# should be migrated from this spec into `PrintAPIs.yaml` (even if we change
+# it a lot!), with the end result being this file is empty and deleted.
+# --------------------------------------------------------------------------------
+#
+
 openapi: 3.0.0
```

Comment:
This is the initial openAPI spec that was added in PR https://github.com/cabinetoffice/eip-ero-print-api/pull/1 
As agreed I have moved it here, and added this comment block